collections/global-surface-water.yaml

```diff
@@ -17,17 +17,17 @@ BandInformation:
         - Name: Description
           Title: Description  
       Rows:
-        - Name: Occurrence
+        - Name: occurrence
```

Comment:
Where did you get those band names from? Looking at "https://services.sentinel-hub.com/api/v1/metadata/collection/byoc-9a525f12-33b6-424e-a0f2-d567eec0f277" I get those names back.

```json
   "bands": [
        {
            "name": "change",
            "sampleType": "UINT8"
        },
        {
            "name": "extent",
            "sampleType": "UINT8"
        },
        {
            "name": "occurrence",
            "sampleType": "UINT8"
        },
        {
            "name": "recurrence",
            "sampleType": "UINT8"
        },
        {
            "name": "seasonality",
            "sampleType": "UINT8"
        },
        {
            "name": "transitions",
            "sampleType": "UINT8"
        }
    ]
```

Reply:
From Nika. Here's the [issue](https://git.sinergise.com/operations/teams/sh-vas/-/issues/243) and the [band info](https://developers.google.com/earth-engine/datasets/catalog/JRC_GSW1_3_GlobalSurfaceWater)

Reply:
@chorng In general lets try to link issues in the PR description. It will make it easier to review the request for the person doing it and to find things again in a couple of month.

The band info you are linking is for the same data but on google earth engine. They also ingested the data to their storage but used different band names. I think we should use here the band names that we also used when adding the product to creodias which are listed by our metadata service.

Reply:
Understood. Will put the issue link in the description next time. The band names are fixed now.